README.md

```diff
@@ -1,16 +1,94 @@
+  - You'll need to contact a team member or a `cisagov` administrator
+- How do I get access to a specific repository?
+  - Most repositories are associated with the [teams](https://docs.github.com/en/organizations/organizing-members-into-teams)
+  that use them, so to get access you can contact github@cisa.gov, a
+  team member, or a `cisagov` administrator to get access
+- How do I make a new repository?
+  - [GitHub's documentation on creating a repository](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-new-repository)
+  - [`cisagov`'s Project Setup guide](https://github.com/cisagov/development-guide/tree/develop/project_setup)
+
+## Policies and content guides ##
+
+- What belongs in `cisagov` versus another GitHub organization?
+  - Here are some questions to ask when considering posting a project:
+    - Does CISA use or develop the software? Is it developed by or for one of
+    the groups or divisions within CISA?
+      - If not, we recommend the authors create their own GitHub organization
+      and post their work there
+    - What source control system is in place currently?
+      - Many source control systems, such as Mercurial and GitLab, can export
+      the entire development history for import into GitHub - this is by far
+      the preferred method
+    - Is the project still under active development or is it in maintenance?
+      - If the project no longer has a team performing maintenance, we
+      recommend the repository be archived to make that clear to people who
+      may want to use it
+- What belongs on [cisa.gov](https://www.cisa.gov) versus on `cisagov`?
+  - The [cisa.gov](https://www.cisa.gov) site is primarily focused at an
+  audience outside of CISA, such as Critical Infrastructure partners or the
+  public
+  - `cisagov` is for both internal and external users, as well as partners. It exists
+  specifically to share projects with the public as well as internal users.
+- Working in the public (dos and don'ts, best practices)
+  - As a best practice, use the `cisagov` organization issue templates and
+  pull request templates. These templates are available by default in
+  all repositories created in the `cisagov` organization.
+  - As a best practice, we require code reviews before merging pull requests.
+  This is done using [branch protection](https://docs.github.com/en/github/administering-a-repository/about-protected-branches).
+- When should I talk to CISA External Affairs (EA)?
+  - Early and often!
+- What is CISA's open source policy?
+  - See [CISA's open source policy here](https://github.com/cisagov/development-guide/blob/develop/open-source-policy/policy.md)
+
+## Feedback and contact ##
+
+Have an idea about how to make these pages better? [File an issue!](https://github.com/cisagov/cisagov/issues/new/choose)
+
+For any repository-specific questions or feedback, please make an issue in
+that repository so the appropriate team will see it.
```

Comment:
Should we also mention here that some of our repositories have [GitHub Discussions](https://docs.github.com/en/discussions) enabled and that is another avenue for questions?

Reply:
The issue-chooser mentions Discussions and Wiki, plus links to this: https://github.com/cisagov/.github/blob/develop/QUESTIONS.md

I'm ambivalent, slightly on the side of leaving it simpler here in the general suggestion. This way the path is obviously "have a question" -> "go to the repository".

Reply:
Good point- I forgot that we mention it in the issue-chooser, so I'm on board with keeping it simple.